components/ip-addresses/kbc-public-ip.json

```diff
@@ -79,6 +79,48 @@
             "vendor": "azure",
             "region": "north-europe",
             "service": "syrup"
+        },
+        {
+            "ipPrefix": "20.82.252.94/32",
+            "vendor": "azure",
+            "region": "north-europe",
+            "service": "syrup"
+        },
+        {
+            "ipPrefix": "20.82.252.129/32",
+            "vendor": "azure",
+            "region": "north-europe",
+            "service": "syrup"
+        },
+        {
+            "ipPrefix": "20.82.252.124/32",
+            "vendor": "azure",
+            "region": "north-europe",
+            "service": "syrup"
+        },
+        {
+            "ipPrefix": "40.127.144.42/32",
+            "vendor": "azure",
+            "region": "north-europe",
+            "service": "queue"
+        },
```

Comment:
tuto ip adresu pozeram ze neni pridana pre syrup service, nemala by tam byt tiez aj pre syrup service?

Reply:
jo to by mela, diky fixnu

Reply:
jo tak je tam i pro syrup, byla tam uz z minula, naopak chybely vsechny ty pro queue. Takze je to ok.